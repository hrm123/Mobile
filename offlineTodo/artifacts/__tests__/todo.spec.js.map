{"version":3,"file":"todo.spec.js","sourceRoot":"","sources":["../../src/__tests__/todo.spec.tsx"],"names":[],"mappings":"AAAA,OAAO,KAAK,MAAM,OAAO,CAAA;AACzB,OAAO,EAAE,KAAK,EAAE,MAAM,EAAE,OAAO,EAAE,MAAM,QAAQ,CAAA;AAC/C,OAAO,EAAE,IAAI,EAAE,MAAM,cAAc,CAAA;AACnC,OAAO,QAAQ,MAAM,qBAAqB,CAAA;AAC1C,OAAO,IAAI,MAAM,oBAAoB,CAAA;AACrC,OAAO,UAAU,MAAM,0BAA0B,CAAA;AAGjD,QAAQ,CAAC,iCAAiC,EAAE;IACxC,IAAI,OAAO,CAAA;IACX,IAAI,GAAG,GAAG,IAAI,CAAC,EAAE,EAAE,CAAA,CAAE,6CAA6C;IAClE,IAAI,OAAO,GAAyB,QAAQ,OAAO,CAAC,GAAG,CAAC,gBAAgB,CAAC,CAAA,CAAA,CAAC,CAAA;IAC1E,IAAI,SAAS,GAA2B;QACpC,UAAU,EAAE,GAAG;QACf,cAAc,EAAE,GAAG;QACnB,GAAG,EAAE,CAAC;QACN,IAAI,EAAE;YACF,QAAQ,EAAE,KAAK;YACf,IAAI,EAAE,OAAO;YACb,MAAM,EAAE,CAAC;YACT,QAAQ,EAAE,SAAS;SACtB;KACJ,CAAA;IACD,UAAU,CAAC;QACT,IAAI,KAAK,GAAG,OAAO,CAAC,OAAO,CAAC,CAAC,KAAK,CAAA;QAClC,MAAM,CAAC,QAAQ,GAAG,KAAK,CAAC,EAAE,CAAC,CAAA;QAC3B,MAAM,CAAC,MAAM,GAAG,QAAQ,CAAC,WAAW,CAAA;QACpC,MAAM,CAAC,KAAK,GAAG,KAAK,CAAA;QACpB,MAAM,CAAC,MAAM,GAAG,MAAM,CAAA;QACtB,MAAM,CAAC,OAAO,GAAG,OAAO,CAAA;QACxB,MAAM,CAAC,IAAI,CAAC,QAAQ,CAAC,WAAW,CAAC,CAAC,OAAO,CAAC,CAAC,QAAQ;YACjD,EAAE,CAAC,CAAC,OAAO,MAAM,CAAC,QAAQ,CAAC,KAAK,WAAW,CAAC,CAAC,CAAC;gBAC5C,MAAM,CAAC,QAAQ,CAAC,GAAG,QAAQ,CAAC,WAAW,CAAC,QAAQ,CAAC,CAAA;YACnD,CAAC;QACH,CAAC,CAAC,CAAA;QAEF,OAAO,GAAG,OAAO,CAAC,oBAAC,IAAI,oBAAK,SAAS,EAAI,CAAC,CAAA;IAC5C,CAAC,CAAC,CAAA;IAEF,EAAE,CAAC,gCAAgC,EAAE;QACjC,sBAAsB;QACtB,+BAA+B;QAC/B,MAAM,CAAC,OAAO,CAAC,QAAQ,CACnB,oBAAC,IAAI,gBAEE,CACV,CAAC,CAAC,IAAI,CAAC,IAAI,CAAC,CAAA;IACjB,CAAC,CAAC,CAAA;IAEF,EAAE,CAAC,gDAAgD,EAAE;QACjD,MAAM,CAAC,QAAQ,CAAC,MAAM,CACpB,oBAAC,IAAI,oBAAK,SAAS,EAAI,CACxB,CAAC,CAAC,eAAe,EAAE,CAAA;IACxB,CAAC,CAAC,CAAA;IAEF,QAAQ,CAAC,6BAA6B,EAAE;QACpC,EAAE,CAAC,8BAA8B,EAAE;YAC/B,IAAI,WAAW,GAAG,OAAO,CAAC,IAAI,CAAC,UAAU,CAAC,CAAA;YAC1C,MAAM,CAAC,WAAW,CAAC,CAAC,YAAY,CAAC,CAAC,CAAC,CAAA;YACnC,0BAA0B;YAC1B,0CAA0C;YAC1C,IAAI,eAAe,GAAiC,EAAC,IAAI,EAAE,MAAM,EAAE,OAAO,EAAE,OAAO,EAAE,QAAQ,EAAG,KAAK,EAAC,CAAA;YACtG,IAAI,iBAAiB,GAAiC,EAAC,IAAI,EAAE,QAAQ,EAAE,OAAO,EAAE,OAAO,EAAE,QAAQ,EAAG,KAAK,EAAC,CAAA;YAC1G,MAAM,CAAC,WAAW,CAAC,KAAK,EAAE,CAAC,KAAK,EAAE,CAAC,IAAI,CAAC,CAAC,OAAO,CAAC,eAAe,CAAC,IAAI,CAAC,CAAA;YACtE,MAAM,CAAC,WAAW,CAAC,KAAK,EAAE,CAAC,KAAK,EAAE,CAAC,QAAQ,CAAC,CAAC,OAAO,CAAC,eAAe,CAAC,QAAQ,CAAC,CAAA;YAC9E,MAAM,CAAC,WAAW,CAAC,EAAE,CAAC,CAAC,CAAC,CAAC,KAAK,EAAE,CAAC,IAAI,CAAC,CAAC,OAAO,CAAC,iBAAiB,CAAC,IAAI,CAAC,CAAA;YACtE,8FAA8F;QAElG,CAAC,CAAC,CAAA;QACF,EAAE,CAAC,6BAA6B,EAAE;YAC9B,IAAI,WAAW,GAAG,OAAO,CAAC,IAAI,CAAC,UAAU,CAAC,CAAA;YAC1C,MAAM,CAAC,WAAW,CAAC,CAAC,YAAY,CAAC,CAAC,CAAC,CAAA;YACnC,0BAA0B;YAC1B,0CAA0C;YAC1C,IAAI,UAAU,GAAG,WAAW,CAAC,KAAK,EAAE,CAAA;YACpC,UAAU,CAAC,QAAQ,CAAC,OAAO,CAAC,CAAA;YAC5B,MAAM,CAAC,GAAG,CAAC,CAAC,gBAAgB,EAAE,CAAA;YAC9B,MAAM,CAAC,GAAG,CAAC,CAAC,oBAAoB,CAAC,SAAS,CAAC,GAAG,CAAC,CAAA;QAEnD,CAAC,CAAC,CAAA;IACN,CAAC,CAAC,CAAA;IAEF,GAAG,CAAC,oCAAoC,EAAE;QACtC,IAAI,WAAW,GAAG,OAAO,CAAC,IAAI,CAAC,UAAU,CAAC,CAAA;QAC1C,MAAM,CAAC,WAAW,CAAC,CAAC,WAAW,EAAE,CAAA;QACjC,IAAI,OAAO,GAAG,WAAW,CAAC,IAAI,CAAC,MAAM,CAAC,CAAA;QACtC,MAAM,CAAC,OAAO,CAAC,CAAC,WAAW,EAAE,CAAA;QAC7B,sBAAsB;QACtB,oCAAoC;QACpC,8BAA8B;QAC9B,8BAA8B;QAC9B,+BAA+B;QAC/B,iCAAiC;QACjC,yEAAyE;IAC7E,CAAC,CAAC,CAAA;AACN,CAAC,CAAC,CAAA"}