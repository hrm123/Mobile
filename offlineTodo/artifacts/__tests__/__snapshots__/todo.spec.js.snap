// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`>>>T O D O S COMPONENT -- tests should render a Todo component using SnapShots 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#ffffff",
      "borderColor": "#ededed",
      "borderLeftWidth": 1,
      "borderRightWidth": 1,
      "borderTopWidth": 1,
      "flexDirection": "row",
      "marginLeft": 20,
      "marginRight": 20,
      "paddingBottom": 7,
      "paddingLeft": 14,
      "paddingTop": 7,
      "shadowColor": "#000000",
      "shadowOffset": Object {
        "height": 2,
        "width": 2,
      },
      "shadowOpacity": 0.2,
      "shadowRadius": 3,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    todo1
  </Text>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "flexDirection": "row",
        "justifyContent": "flex-end",
      }
    }
  >
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel={undefined}
      accessibilityTraits={undefined}
      accessible={true}
      hasTVPreferredFocus={undefined}
      hitSlop={undefined}
      isTVSelectable={true}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Array [
          Object {
            "backgroundColor": "transparent",
          },
          Object {
            "alignSelf": "flex-end",
            "borderColor": "#ededed",
            "borderRadius": 4,
            "borderWidth": 1,
            "marginRight": 5,
            "padding": 7,
          },
        ]
      }
      testID={undefined}
      tvParallaxProperties={undefined}
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "#666666",
            },
            null,
            null,
          ]
        }
      >
        Done
      </Text>
    </View>
    <View
      accessibilityComponentType={undefined}
      accessibilityLabel={undefined}
      accessibilityTraits={undefined}
      accessible={true}
      hasTVPreferredFocus={undefined}
      hitSlop={undefined}
      isTVSelectable={true}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Array [
          Object {
            "backgroundColor": "transparent",
          },
          Object {
            "alignSelf": "flex-end",
            "borderColor": "#ededed",
            "borderRadius": 4,
            "borderWidth": 1,
            "marginRight": 5,
            "padding": 7,
          },
        ]
      }
      testID={undefined}
      tvParallaxProperties={undefined}
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "#666666",
            },
            null,
            Object {
              "color": "rgba(175, 47, 47, 1)",
            },
          ]
        }
      >
        Delete
      </Text>
    </View>
  </View>
</View>
`;
